doc/book/transport/smtp-authentication.md

```diff
@@ -122,3 +126,64 @@ $options   = new SmtpOptions([
 ]);
 $transport->setOptions($options);
 ```
+
+### SMTP Transport Usage for servers with reuse time limit
+
+By default every `Zend\Mail\Protocol\Smtp\*` class try to disconnet from the
+STMP server by sending a `QUIT` command and expecting a `221`
+(_Service closing transmission channel_) code response.
+This is done automatically at object `__destruct`ion, and can generate errors
+with SMTP servers like [Posftix](http://www.postfix.org/postconf.5.html#smtp_connection_reuse_time_limit)
+that implement a reuse time limit:
+
+```php
+// [...]
+$transport->send($message);
+
+var_dump('E-mail sent');
+sleep(305);
+var_dump('Soon to exit...');
+exit;
+
+// E-mail sent
+// Soon to exit...
+// Notice: fwrite(): send of 6 bytes failed with errno=32 Broken pipe in ./zend-mail/src/Protocol/AbstractProtocol.php on line 255
+// Fatal error: Uncaught Zend\Mail\Protocol\Exception\RuntimeException: Could not read from 127.0.0.1 in ./zend-mail/src/Protocol/AbstractProtocol.php:301
```

Comment:
The error is intentionally reported so it is easily searchable through serve engines